cmd/ctr/commands/run/run_unix.go

```diff
@@ -61,7 +63,20 @@ func NewContainer(ctx gocontext.Context, client *containerd.Client, context *cli
 		opts = append(opts, withMounts(context))
 
 		if context.Bool("rootfs") {
-			opts = append(opts, oci.WithRootFSPath(ref))
+			rootfs := ref;
+			if !filepath.IsAbs(rootfs) {
```

Comment:
I believe you can just call

```go
rootfs, err := filepath.Abs(ref)
```

Reply:
Great! This code is from runc. With your opinion, I think runc may need to change it's code.

Reply:
https://github.com/opencontainers/runc/blob/master/libcontainer/specconv/spec_linux.go#156